stubs/astropy/astropy/units/equivalencies.pyi

```diff
@@ -0,0 +1,29 @@
+from _typeshed import Incomplete
+from collections import UserList
+
+class Equivalency(UserList[Incomplete]):
+    data: Incomplete
+    name: Incomplete
+    kwargs: Incomplete
+    def __init__(self, equiv_list, name: str = ..., kwargs: Incomplete | None = ...) -> None: ...
+    def __add__(self, other): ...
+    def __eq__(self, other): ...
+
+def dimensionless_angles(): ...
```

Comment:
Doesn't have to be in this diff but it looks like all these functions return `Equivalency`.